.github/workflows/create_release_draft.yml

```
@@ -16,89 +16,75 @@ on:
       - .github/**
       - utils/**
 
+env:
+  OUT_DIR: ${{ github.workspace }}/../grass_outdir
+  GRASS: grass-${{ github.ref_name }}
+permissions:
+  contents: write
 jobs:
   build:
-    runs-on: ubuntu-22.04
-
+    name: Package and create release draft
+    runs-on: ubuntu-latest
```

```suggestion
    name: Package and create release draft
    runs-on: ubuntu-latest
    permissions:
      contents: write
```

Limiting the elevated permissions to the only job that needs.